applications/ParticleMechanicsApplication/README.md

```diff
@@ -67,9 +68,10 @@ The following features are currently available and subject to development within
         * Modified Cam-Clay - finite strain, plane strain, axis-symmetric, and 3D
 
 - Strategies and schemes:
-    * Implicit - Newmark prediction and correction scheme
+    * Implicit - Newmark prediction and correction scheme for static, quasi-static, and dynamic problems
```

Comment:
dynamic -> yet only implicit

Reply:
It is indeed written explicitly in the front before the dash :smile:

Reply:
haha but then this line is very weird...
because it is static quasi static and implicit dynamic

Reply:
Okay, let's make it that way.

Reply:
we'll keep this way as have been discussed with @antonialarese